sdk/src/main/java/io/opentdf/platform/sdk/Config.java

```diff
@@ -16,6 +16,7 @@ public class Config {
     public static final int TDF3_KEY_SIZE = 2048;
     public static final int DEFAULT_SEGMENT_SIZE = 2 * 1024 * 1024; // 2mb
     public static final String KAS_PUBLIC_KEY_PATH = "/kas_public_key";
+    public static final String DEFAULT_MIME_TYPE = "application/octet-stream";
```

Comment:
I think the other clients default to no mime type (the field is dropped)

Reply:
CLI defaults to `application/octet-stream` which was modeled from MDM web docs. We can change this if needed. https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

> application/octet-stream is the default value for all other cases. An unknown file type should use this type. Browsers are particularly careful when manipulating these files to protect users from software vulnerabilities and possible dangerous behavior.

Reply:
https://github.com/opentdf/platform/blob/main/sdk/tdf.go#L252